### PR TITLE
Added text field for version edition

### PR DESCRIPTION
**Pull request by BryceMarshall**

### CODE REVIEW

**Comment by n-a-t-e** [comment]
Looks good, here are some other steps:

- add `edition` field to record_json_to_yaml.py
- update recordToERDDAP.js using `product_version` https://wiki.esipfed.org/Attribute_Convention_for_Data_Discovery_1-3#product_version . Sound good @JessyBarrette ?
- add the xml to main.j2 in the metadata-xml repo (see #208 )
- try to add `edition` to citation preview generated by citation-js

**Comment by n-a-t-e** [comment]
Fixes #208

File: src/components/FormComponents/ApaPreview.jsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ export function generateCitation(record,language, format) {
       issued: { "date-parts": [[datePublished || created]] },
       publisher: publishers.join(", "),
       DOI: datasetIdentifier.replace(/https?:\/\/doi\.org\//, ""),
+      version: record.edition,
```
Comment (BryceMarshall) [inline src/components/FormComponents/ApaPreview.jsx:52]: any issue with wording consistency here? all references to the field are for `edition` but the citation indicates `version`.

Comment (n-a-t-e) [reply src/components/FormComponents/ApaPreview.jsx:52]: its fine for now, we will review what the citation looks like and can change if needed. It was suggested that we prefix version with a  'v' if the library doesn't do that already

File: src/utils/recordToERDDAP.js [hunk 2]
```diff
@@ -48,6 +48,7 @@ function recordToERDDAP(record) {
       .filter((e) => e)
       .join(),
     project: record.projects?.join(),
+    project_version: record.edition,
```
Comment (n-a-t-e) [inline src/utils/recordToERDDAP.js:51]: should be `product_version`

**Pull request merged**